src/main/java/core/basesyntax/behaviour/Geometrical.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.behaviour;
+
+public interface Geometrical {
```

Comment:
Consider renaming the interface to 'Geometric' to better match the task description and common naming conventions.